docs/magicc_flags.rst

```diff
@@ -0,0 +1,215 @@
+            cfg = update_cfg_from_tuningmodel_like_magicc(
+                cfg,
+                cfg[namelist_to_derive]["file_tuningmodel_1"]
+            )
+
+        for i in range(2, 50):
+            key_to_check = "file_tuningmodel_{}".format(i)
+            if key in cfg[namelist_to_derive]:
+                cfg = update_cfg_from_tuningmodel_like_magicc(
+                    cfg,
+                    cfg[namelist_to_derive][key]
+                )
+
+        return cfg
+
+    # Example usage
+    run_dir = "/here/there/MAGICC/run"
+    namelist_to_derive = "all_cfgs"
+    output_file = "./somewhere/else/example.cfg"
+    derive_final_cfg(run_dir, namelist_to_derive).write(output_file)
+
+
+To avoid any really unexpected, silent surprises, we want the pymagicc ``.CFG`` file,
+``MAGTUNE_PYMAGICC.CFG`` to overwrite everything else. To save trying to debug
+extremely tricky overwriting setups, we enforce simple configurations in Pymagicc and
+raise ``ValueError`` if they are not adhered to.
+
+
+Clash with scenario flags
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These 'conventions' become more confusing when we compare to what happens with the
+emission scenario files.
+
+In MAGICC6 it's simple, there's only one emissions scenario file and it goes in
+``FILE_EMISSIONSCENARIO=DFGH``. MAGICC then looks for files that match ``DFGH`` and
+``DFGH.SCEN``.
+
+In MAGICC7, there are now multiple ``FILE_EMISSCEN_X`` flags (note the shift from
+``FILE_EMISSIONSCENARIO``). The values found in the file specified in each
+``FILE_EMISSCEN_X`` flag overwrite any previously read in values.
+
+*Gotcha:* The first emissions scenario file is specified by ``FILE_EMISSCEN`` (without
+the number), which matches the MAGICC6 convention for ``FILE_TUNINGMODEL`` but
+contradicts the MAGICC7 convention for ``FILE_TUNINGMODEL``.
+
+*Gotcha:* Say we have ``FILE_EMISSCEN=DFGH``, MAGICC7 looks for files that match
+``DFGH``, then ``DFGH.SCEN7`` and then ``DFGH.SCEN`` (in that order). Hence the first
+emissions scenario can be SCEN7, or SCEN, with preference being given to SCEN7 files
+if there are two scenario files with the same stem (i.e. ``RCP26.SCEN7`` is chosen
+before ``RCP26.SCEN`` if ``FILE_EMISSCEN=RCP26``).
+
+*Gotcha:* In MAGICC7, only the first ``FILE_EMISSCEN`` can be a ``SCEN`` file (in
+MAGICC6 there can only be ``SCEN`` files so this isn't an issue). All other
+``FILE_EMISSCEN_X`` files can only be SCEN7 files. The rationale here (I think) is
+that SCEN files don't contain easy to read metadata hence overwriting with them is
+difficult/dangerous.
+
+*Gotcha:* If you set ``FILE_EMISSCEN_X=NONE`` then MAGICC will just move on to the
+next ``FILE_EMISSCEN_X`` flag. However, from above it's clear that if you set
+``FILE_TUNINGMODEL_X=NONE``, MAGICC will look for ``MAGTUNE_NONE.CFG``, not find it
+and blow up. Hence there's a direct contradiction there too.
+
+*Gotcha:* Of course the final part is that each ``.CFG`` file can overwrite the
```

Comment:
Maybe Pymagicc should only support SCEN7 and have a conversion function (if possible)?

Reply:
That's an option although at the moment it basically only supports taking in dataframes (and then does the conversions internally, or will do after #168) so it's kind of a non-issue?

Reply:
Ok, seems to be non-issue then ...